Makefile

```
@@ -324,6 +324,16 @@ gitalias-update: ## Download latest gitalias.txt from upstream.
 	@./scripts/update-gitalias.sh
 	@echo "✅ gitalias.txt updated"
 
+.PHONY: moshi-update
+moshi-update: ## Sync moshi-hook generated configs from live to dotfiles.
```

checkmake (0.3.2)

[warning] Target body for "moshi-update" exceeds allowed length of 5 lines (7).

(maxbodylength)